lib/configfiles/rhel8.xml

```diff
@@ -0,0 +1,2448 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
any specific changes from 7 to 8?

Reply:
none currently,
however RHEL8 is running postfix 3.3 and RHEL7 is running 2.10
By default postfix 3.3 runs in backwards compatibility mode and i have tested it to work with that.
if i get around to it i'll see if the postfix 3.3 config can be optimized and thus changing it for rhel 8.